internal/db/repos/user.go

```
@@ -0,0 +1,66 @@
+	"fmt"
+
+	"github.com/celestiaorg/talis/internal/db/models"
+	"gorm.io/gorm"
+)
+
+// UserRepository handles database operations for user entities
+type UserRepository struct {
+	db *gorm.DB
+}
+
+// UserRepository handles database operations for user entities
```

golangci-lint (1.64.8)

[warning] exported: comment on exported function NewUserRepository should be of the form "NewUserRepository ..."

(revive)